README.md

```diff
@@ -128,6 +101,16 @@ server.listen(server.get('port'), () => {
 });
 ```
 
+## Default components
+
+The `Oy` namespace exposes the following components validated against email best practices: 
+
+```
+Table TBody TR TD Img A
+```
+
+If you want to circumvent this validation, you can use `Oy.Element` and pass the `tagName` prop to implement your own validated element.
```

Comment:
The problem with that is you lose the preservation of HTML4 attributes. I'll add a sentence clarifying that. Thanks!